mysite/code/HomePage.php

```diff
@@ -1,9 +0,0 @@
-<?php
-
-class HomePage extends BaseHomePage {
-	private static $description = 'Entry to the site. Has carousel, featured panels, quicklinks and news summaries';
-}
-
-class HomePage_Controller extends BaseHomePage_Controller {
-	
-}
```

Comment:
Since we won't be creating a `cwp/cwp-recipe-cms`, but inlining the `cwp/cwp` module along with `silverstripe/recipe-cms`, we need to keep these two classes here. `cwp/cwp` depends on them for home page functionality to work.